src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonValueOfJsonPrimitive.cs

```diff
@@ -35,7 +43,7 @@ internal static JsonValue CreatePrimitiveValue(ref Utf8JsonReader reader, JsonNo
                     }
 
                     quotedStringValue[0] = quotedStringValue[quotedStringValue.Length - 1] = JsonConstants.Quote;
-                    return new JsonValueOfJsonString(quotedStringValue, reader.ValueIsEscaped, options);
+                    return new JsonValueOfJsonString(quotedStringValue, options);
```

Comment:
Why would we need to also store the quotes? Couldn't we just store a plain UTF-8 string?